apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -3051,7 +3051,8 @@
   "Livechat_agents": "Omnichannel agents",
   "Livechat_Agents": "Agents",
   "Livechat_allow_manual_on_hold": "Allow agents to manually place chat On Hold",
-  "Livechat_allow_manual_on_hold_Description": "If enabled, the agent will get a new option to place a chat On Hold, provided the agent has sent the last message",
+  "Livechat_allow_manual_on_hold_upon_agent_engagement_only": "Allow agents to manually place chat On Hold, provided the agent has sent the last message",
+  "Livechat_allow_manual_on_hold_upon_agent_engagement_only_Description": "If enabled, the agent will get a new option to place a chat On Hold, provided the agent has sent the last message, otherwise Agents are able to place chats onHold irrespective",
```

Comment:
This description is not quite true. When setting is enabled, we get the same behavior we currently have, so it's not a "new option".